### DISCUSSION TITLE
Упрощенная реклама

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#538] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kyper/32/363_2.png) [Kyper](https://forum.octothorp.team/u/Kyper) ([#4243](https://forum.octothorp.team/p/4243))

Такая идея, у мэра будет возможность помечать на карте всякие забегаловки, магазины оружия и.т.д. Как это будет работать?

1. Человек ,имеющий бизнес, приходит в мэрию, заранее выкупив помещение и расставив оборудование внутри.
2. Далее он подходит к мэру и нажимает на него ПКМ через “C”, после чего появляется новая вкладка под названием “Запросить рекламу на карте” или что-то типо того.
3. После нажатия на нее у игрока будет открываться карта, на которой он сможет видеть только свои помещения, нажав на которые откроется список, в котором обязательно нужно будет указать вид деятельности, а остальные будут необязательные строки, например: Имя, номер, график работы и слова от самого заказчика рекламы.
4. После заполнения всего у мэра появляется вкладка слева ( как лотерея или голосование )
5. После того, как мэр примет заказ у него тоже откроется карта, где он сможет разглядеть, где именно будет это место на карте, а потом просто нажав “Принять” располагает на карте эту метку, которую видят все и на своей карте.

Также добавить общее ограничение на такие метки после 5-6 ,уже поставленных, меток на карте

Если обычный игрок просто наведет на нее курсор, то высветится вся информация про это заведение включая все что писал заказчик при оформлении рекламы.

Это поможет игрокам понимать куда им идти для удовлетворения их надобностей не выходя из дома.

( Еще добавить сохранение всех записей при оформлении рекламы, например: если мэр проигнорировал или случайно отклонил заказ, чтобы заказчику по новой не приходилось все записывать. )

Упростит рекламу, появятся больше посетителей и просто добавит чуточку разнообразия

- Да, стоит добавить
- Нет, не стоит добавлять

0голосов

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/tecbill/32/660_2.png) [TecBill](https://forum.octothorp.team/u/TecBill) ([#4279](https://forum.octothorp.team/p/4279))


Интересно, но трудно

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/bnod/32/371_2.png) [BNOD](https://forum.octothorp.team/u/BNOD) ([#4288](https://forum.octothorp.team/p/4288))


Интересная система, но не лучше сделать это логичнее через телефон? Типо было бы неплохо помечать места, даже когда мэра нету, например через тот же телефон, платя при этом небольшую сумму денег, главное, чтобы это не переросло в бессмысленный спам этими же метками.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/de_kuyash/32/958_2.png) [De_Kuyash](https://forum.octothorp.team/u/De_Kuyash) ([#4345](https://forum.octothorp.team/p/4345))


Жду не дождусь +100500 меток на Рузвельт с “РП” продажей глоков с отмычками.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kyper/32/363_2.png) [Kyper](https://forum.octothorp.team/u/Kyper) ([#4353](https://forum.octothorp.team/p/4353))


Вот как раз таки из-за спама и не стоит. Если мэр адекватный, то он будет понимать, когда нужно прекратить. А так люди будут сходить с ума с этими метками

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kyper/32/363_2.png) [Kyper](https://forum.octothorp.team/u/Kyper) ([#4365](https://forum.octothorp.team/p/4365))


Ну это будет недоступно для профессий, которые имеют совсем другую роль, не связанную с бизнесом.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/пончик/32/1539_2.png) [ПоНчИк](https://forum.octothorp.team/u/ПоНчИк) ([#4368](https://forum.octothorp.team/p/4368))


очень интересная и очень полезная идея. Конечно, механика сложная, но вполне реализуемая. Естественно из всего предложенного можно изменить пару аспектов, например что размещение меток можно делать через телефон. а если нету мэра? если игрок вышел? если ложная информация? если бизнес закрыт? Я определённо **ЗА**, но доработать необходимо!

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kyper/32/363_2.png) [Kyper](https://forum.octothorp.team/u/Kyper) ([#4372](https://forum.octothorp.team/p/4372))


Как я уже сказал должны быть ограничения, дабы избежать спама, а если заказать такую рекламу можно будет просто через телефон, то игроки просто напросто за пару сек разберут все слоты . Убирать данную метку смогут только мэр и заказчик, а что, если мэра нет, а заказчик тупо не открывает своё заведение или что-нибудь ещё, что тогда делать? Поэтому лучше дать эту возможность только мэру. Доработать где-то стоит, но в этом плане думаю не надо.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kustik/32/467_2.png) [Kustik](https://forum.octothorp.team/u/Kustik) ([#4736](https://forum.octothorp.team/p/4736))


Неплохое предложение, но я бы добавил пару бы уточнений для такой рекламы:

1. Стоимость такой Метки - от 5000₽ минимум;
2. Заказывать надо через телефон, но подтверждать это должен Мэр;
3. К/Д использования - 30 минут;
4. 1 человек - 1 Метка на карте;
5. Сделать отдельную от F4 меню карту с этими Метками, чтобы всем было удобно;
6. Минимальное количество часов для использования - 5-10 Часов;
7. <s>Можно будет менять иконки Метки;</s>
8. Ограничение по названию Метки - 32 символа;
9. Убрать красный мигающий круг у таких Меток;

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kolliderre/32/1363_2.png) [KolliDerre](https://forum.octothorp.team/u/KolliDerre) ([#5367](https://forum.octothorp.team/p/5367))


Не сказал бы, если понять то будет очень легко

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kolliderre/32/1363_2.png) [KolliDerre](https://forum.octothorp.team/u/KolliDerre) ([#5368](https://forum.octothorp.team/p/5368))


Можно просто напросто создать галочку в стиле “Отображать предприятия”

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/wani4ka/32/27_2.png) [Wani4ka](https://forum.octothorp.team/u/Wani4ka) ([#5801](https://forum.octothorp.team/p/5801))


Слишком много механа, как по мне. Я бы просто разрешил мэру размещать до, скажем, 10 меток различных заведений на карте, которые бы видели все. А за такую услугу он может запросить деньги, налоги, по блату, натурой расплатиться - как угодно.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kyper/32/363_2.png) [Kyper](https://forum.octothorp.team/u/Kyper) ([#5848](https://forum.octothorp.team/p/5848))


Это текст так написан, механа тут очень мало

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/zhilichni/32/641_2.png) [zhilichni](https://forum.octothorp.team/u/zhilichni) ([#7440](https://forum.octothorp.team/p/7440))


Я бы хотел такую возможность, это было бы очень удобно и не надо будет десятками минутами искать нужное тебе заведение, **ЗА**

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/pinqfox/32/754_2.png) [pinqfox](https://forum.octothorp.team/u/pinqfox) ([#8560](https://forum.octothorp.team/p/8560))


Это конечно хорошо, но что будет, когда владелец предприятия выйдет из игры, или съедет с места, при этом не предупредив мэра? Эта метка останется на том же самом месте, туда заедет другой человек, например если у первого была там токарня, то у второго ресторан, метка останется и люди будут туда приходить каждые 5 минут в поисках токаря.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kyper/32/363_2.png) [Kyper](https://forum.octothorp.team/u/Kyper) ([#8934](https://forum.octothorp.team/p/8934))


Я написал, что заказчике при оформлении заказа будет видеть на карте свое место жительства, соответственно сервер будет распознавать, когда он там живет, а когда его там нет.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/blackmailer/32/2271_2.png) [blackmailer](https://forum.octothorp.team/u/blackmailer) ([#9578](https://forum.octothorp.team/p/9578))


Такие нововведения только повысят РП жизнь на сервере! Я только **ЗА**!
 Правда, чтобы реализовать такое, нужно грамотно доработать все мелочи, но идея **достойна** внимания. ![:ok_hand:](https://forum.octothorp.team/images/emoji/twitter/ok_hand.png?v=10 ":ok_hand:")

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/d0nbar0n/32/2122_2.png) [D0nBar0n](https://forum.octothorp.team/u/D0nBar0n) ([#9636](https://forum.octothorp.team/p/9636))


Конечно сложно в реализации, но задумка очень интересная, продвигает работу бизнесом и взаимодействие людей, **я ЗА!**

**Comment by LENFYS** [comment]
За

**Comment by msmr-dev** [comment]
го
